ansible-ipi-install/roles/installer/tasks/main.yml

```diff
@@ -2,6 +2,10 @@
 # tasks file for installer
 - include_tasks: 10_get_oc.yml 
 - include_tasks: 20_extract_installer.yml
+- include_tasks: 23_rhcos_image_paths.yml
+  when: ((release_version[0]|int == 4) and (release_version[2]|int <= 3)) or cache_enabled == true
```

Comment:
just confirming here the behavior. So we expect 23_* yml to run when version is 4.3 or lower or cache is true

Reply:
Yes, 23_* runs if the cache is enabled or if the version is 4.3.  4.3 needs 23_* regardless of cache due to the RHCOS image paths needed in metal3-config.yaml.